plugins/src/main/java/org/smartregister/fhir/gateway/plugins/PractitionerDetailsEndpointHelper.java

```diff
@@ -43,7 +43,7 @@ public class PractitionerDetailsEndpointHelper {
     public static final Bundle EMPTY_BUNDLE = new Bundle();
     private final IGenericClient r4FHIRClient;
 
-    private final LocationHierarchyEndpointHelper locationHierarchyEndpointHelper;
+    private static LocationHierarchyEndpointHelper locationHierarchyEndpointHelper;
```

Comment:
This field doesn't seem to be used anywhere else in this class. Let's remove this field and instantiate it at the only point of usage.

**Note:** For this to work you will need to refactor `IGenericClient getFhirClientForR4() `method to static so that it can be used in the constructor of `LocationHierarchyEndpointHelper`

Reply:
It is used here https://github.com/onaio/fhir-gateway-extension/blob/related_entity_location_sync_strategy/plugins/src/main/java/org/smartregister/fhir/gateway/plugins/PractitionerDetailsEndpointHelper.java#L50